source/use-cases/local-master-key.txt

```diff
@@ -0,0 +1,46 @@
+A. Create a Local Master Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
I think it would be good to provide some emphasis on "envelope encryption" since it's a term we're introducing.

Reply:
Sounds good to me, will update.